tests/buyProduct_test.js

```diff
@@ -22,20 +27,23 @@ let addressees = {
     city: "Kyiv",
 };
 
+const LinksGetter = require('../helpers/productLinksGetter.js');
+let productLinks3 = LinksGetter.getLinks();
+console.log(productLinks3);
+
 Feature('buy product');
 
-Scenario('buy product', async ({ I, productPage, checkoutPage }) => {
+Before(({ I }) => {
     I.login(loginUser);
-    I.openProductPage();
+});
+
+Data(productLinks3).Scenario('buy product', async ({ I, productPage, checkoutPage, current }) => {
+    console.log(current.link)
+    I.openProductPage(current.link);
```

Comment:
use `I.amOnPage()` method.
remove `openProductPage()` since it has only one instruction.

Reply:
> use `I.amOnPage()` method. remove `openProductPage()` since it has only one instruction.

 my method openProductPage is amOnPage
 
 openProductPage(url) {
      this.amOnPage(url);
    },